tests/phpunit/tests/media.php

```diff
@@ -3973,6 +3973,54 @@ public function test_wp_generate_attachment_metadata_doesnt_generate_sizes_for_1
 		);
 	}
 
+	/**
+	 * @ticket 58212
+	 * @covers ::wp_get_attachment_image()
+	 */
+	public function test_wp_get_attachment_image_context_filter_default() {
+		$last_context = '';
+		add_filter(
+			'wp_get_attachment_image_context',
+			function( $context ) use ( &$last_context ) {
+				$last_context = $context;
+				return $context;
```

Comment:
```suggestion
				return $last_context;
```

Reply:
Why is that change useful? To me it seems either is fine. Conceptually speaking, I find it more intuitive to return `$context` as that is also the parameter and returning that indicates that we're not overwriting it in any way.

Reply:
Right, the point of this filter is just to capture whatever the last context was, not to mutate it in any way.